docs/vm/live-migration.md

```diff
@@ -26,6 +26,22 @@ Live migration means moving a virtual machine to a different host without downti
 
 :::
 
+## How Migration Works
+
+Each node has multiple CPU models that are labeled with different keys.
+
+- Primary CPU model: `host-model-cpu.node.kubevirt.io/{cpu-model}`
+- Supported CPU models: `cpu-model.node.kubevirt.io/{cpu-model}`
+- Supported CPU models for migration: `cpu-model-migration.node.kubevirt.io/{cpu-model}`
+
+During live migration, the system checks the value of `spec.domain.cpu.model` in the VirtualMachineInstance (VMI) CR, which is derived from `spec.template.spec.domain.cpu.model` in the VirtualMachine (VM) CR. If the value of `spec.template.spec.domain.cpu.model` is not set, the system uses the default value `host-model`.
+
+When `host-model` is used, the process fetches the value of the primary CPU model and fills `spec.NodeSelectors` of the newly created pod with the label `cpu-model-migration.node.kubevirt.io/{cpu-model}`. 
+
+Besides, you can customize the CPU model in `spec.domain.cpu.model` instead of using default `host-model`. For example, if the CPU model is `XYZ`, the process fills `spec.NodeSelectors` of the newly created pod with the label `cpu-model.node.kubevirt.io/XYZ`.
```

Comment:
Here, it's little different between `spec.domain.cpu.model: host-model` and `spec.domain.cpu.model: XYZ`.  So, I added one paragraph to describe that.